pkg/core/object/fmt.go

```diff
@@ -191,6 +241,32 @@ func (v *FormatValidator) ValidateContent(o *object.Object) (ContentMeta, error)
 	switch o.Type() {
 	case object.TypeRegular:
 		// ignore regular objects, they do not need payload formatting
+	case object.TypeLink:
+		if len(o.Payload()) == 0 {
+			return ContentMeta{}, fmt.Errorf("(%T) empty payload in the link object", v)
+		}
+
+		firstObjID, set := o.FirstID()
+		if !set {
+			return ContentMeta{}, errors.New("link object does not have first object ID")
+		}
+
+		cnr, set := o.ContainerID()
+		if !set {
+			return ContentMeta{}, errors.New("link object does not have container ID")
+		}
+
+		var testLink object.Link
+
+		err := o.ReadLink(&testLink)
+		if err != nil {
+			return ContentMeta{}, fmt.Errorf("reading link object's payload: %w", err)
+		}
+
+		err = v.sv.VerifySplit(context.Background(), cnr, firstObjID, testLink.Objects())
```

Comment:
why ctx is parameterized if we pass `context.Background` only?

Reply:
i am dreaming of having context in every multithread/network/slice-parameterised operations. `SplitVerifier` wants it but `FormatValidator` does not have it yet

Reply:
this is a good dream, so lets accept context prm then